src/anndata/_core/merge.py

```diff
@@ -757,6 +757,7 @@ def np_bool_to_pd_bool_array(df: pd.DataFrame):
     return df
 
 
+# TODO: concat for xarray
```

Comment:
This would be nice to have with this PR. What are your thoughts on this @ilan-gold?

Reply:
Yea......I agree......

Reply:
Ok, sorry, I remember now.  I looked into this...it's a lot of work...but maybe I'll try my hand at it....sometimes you get fatigue just from how daunting something seems

Reply:
Ah @dcherian I am actually almost 100% sure this is already implemented in `xarray`, but we need to get the functionality here!  I think it's all good on your end but I will let you know if we need anything!